docs/mast/mast.rst

```diff
@@ -181,6 +180,111 @@ To get a table of metadata associated with observation or product lists use the
+and 'hst' mission.
+
+.. code-block:: python
+
+                >>> from astroquery.mast.missions import MastMissions
+                >>> missions = MastMissions()
+                >>> missions.mission
+                'hst'
+                >>> missions.service
+                'search'
+
+The missions object can be used to search meta data using region coordinates. the kwargs can be
+used to specify output characteristics like selec_cols and sort_by and conditions that filter on 
+values like proposal id, pi last name etc. The available column names to filter and
+their descriptions can be found here https://vao.stsci.edu/missionmast/tapservice.aspx/tables#folder38
+
+For a cone search, select_cols would always include ang_sep, sci_data_set_name, search_key and 
+search_position, in addition to the columns specified using select_cols.
+
+for a non cone search, select_cols would always include search_key and sci_data_set_name.
+
+.. code-block:: python
+
+                >>> from astroquery.mast.missions import MastMissions
+                >>> from astropy.coordinates import SkyCoord
+                >>> missions = MastMissions()
+                >>> regionCoords = SkyCoord(210.80227, 54.34895, unit=('deg', 'deg'))
+                >>> results = missions.query_region(regionCoords, 3, sci_pep_id=12556, select_cols=["sci_stop_time", "sci_targname", "sci_start_time", "sci_status"], sort_by=['sci_targname'])
+                >>> results[:5]
+                <Table masked=True length=5>
+                sci_status   sci_targname   sci_data_set_name       ang_sep        sci_pep_id     search_pos     sci_pi_last_name          search_key        
+                   str6         str16              str9              str20           int64          str18              str6                  str27           
+                ---------- ---------------- ----------------- -------------------- ---------- ------------------ ---------------- ---------------------------
+                    PUBLIC NUCLEUS+HODGE602         OBQU010H0 0.017460048037303017      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010H0
+                    PUBLIC NUCLEUS+HODGE602         OBQU01050 0.017460048037303017      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU01050
+                    PUBLIC NUCLEUS+HODGE602         OBQU01030 0.022143836477276503      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU01030
+                    PUBLIC NUCLEUS+HODGE602         OBQU010F0 0.022143836477276503      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010F0
+                    PUBLIC NUCLEUS+HODGE602         OBQU010J0  0.04381046755938432      12556 210.80227 54.34895           GORDON 210.80227 54.34895OBQU010J0
+
+
+for paging through the results, offset and limit can be used to specify the starting record and the number
+of returned records. the default values for offset and limit is 0 and 5000 respectively.
```

Comment:
Is this example needed? How often do you expect users to want to limit the results this way?  

What happens if a query returns more than 5000 results? Is there a warning?

Reply:
No, there is no warning. If the limit is not set then it uses a default of 5000 and even if the result set contains more than 5000, it only returns the first 5000.

Reply:
In that case there should be a warning given when the maximum number of results is returned so the user knows they might not be seeing all the results. You can see what the HST catalogue call does for how to implement this.